PopulateTools/gobierto
# CMS page implementation

Connects to #928 

### What does this PR do?
- [x] A user can visit a section, and she will be shown the first page of the section according to its hierarchy.
- [x] A user can visit any page from a section.
- [x] When a user visits a page, she will be shown:
- [x] The menu from the section, with the page's branch expanded, and the current page selected
- [x] The breadcrumb, which is formed by: Section name + page hierarchy (ignoring root element)
- [x] The content of the page
- [x] A box with attachments, if they exist

### How should this be manually tested?
Editing a section in the back and see section in the front.
Do not use this feature (https://github.com/PopulateTools/gobierto/pull/1017) yet

### New issue

https://github.com/PopulateTools/gobierto/issues/1117
